docs/design/ovs-pipeline.md

```diff
@@ -1,128 +1,218 @@
+
+## OVS Ct Mark
+
+We use some bits of the OVS connection tracking mark to carry information throughout the pipeline. To enhance usability,
+we assign friendly names to the bits we use.
+
+| Field Range | Field Name            | Ct Mark Value | Ct Mark Name       | Description                                                     |
+|-------------|-----------------------|---------------|--------------------|-----------------------------------------------------------------|
+| bits 0-3    | ConnSourceCTMarkField | 0b0010        | FromGatewayCTMark  | Connection source is Antrea gateway port.                       |
+|             |                       | 0b0101        | FromBridgeCTMark   | Connection source is local bridge port.                         |
+| bit 4       |                       | 0b1           | ServiceCTMark      | Connection is for Service.                                      |
+|             |                       | 0b0           | NotServiceCTMark   | Connection is not for Service.                                  |
+| bit 5       |                       | 0b1           | ConnSNATCTMark     | SNAT'd connection for Service.                                  |
+| bit 6       |                       | 0b1           | HairpinCTMark      | Hair-pin connection.                                            |
+| bit 7       |                       | 0b1           | L7NPRedirectCTMark | Connection should be redirected to an application-aware engine. |
+
+## OVS Ct Label
+
+We use some bits of OVS connection tracking label to carry information throughout the pipeline. To enhance usability,
+we assign friendly names to the bits we use.
+
+| Field Range | Field Name            | Description                        |
+|-------------|-----------------------|------------------------------------|
+| bits 0-31   | IngressRuleCTLabel    | Ingress rule ID.                   |
+| bits 32-63  | EgressRuleCTLabel     | Egress rule ID.                    |
+| bits 64-75  | L7NPRuleVlanIDCTLabel | VLAN ID for L7 NetworkPolicy rule. |
+
+## OVS Ct Zone
+
+We use some OVS connection tracking zones to isolate connection tracking rules. To enhance usability, we assign friendly
+names to the ct zones.
+
+| Zone ID | Zone Name    | Description                                        |
+|---------|--------------|----------------------------------------------------|
+| 65520   | CtZone       | Tracking IPv4 connections that don't require SNAT. |
+| 65521   | SNATCtZone   | Tracking IPv4 connections that require SNAT.       |
+
+## Kubernetes NetworkPolicy Implementation
+
+Several tables of the pipeline are dedicated to [Kubernetes
+NetworkPolicy](https://kubernetes.io/docs/concepts/services-networking/network-policies/) implementation (table
+[EgressRule], [EgressDefault], [IngressRule] and [IngressDefault]).
+
+The Antrea implementation of Kubernetes NetworkPolicy, including the communication channel between the Antrea Controller
```

Comment:
This paragraph is confusing to me. Did you mean the implementation of `Antrea-native Policy` is described in another document?

Reply:
I copied it from the original doc. I found this is not ver relevant to the pipeline doc. I'll remove it.